src/pyscript/plugins/create.py

```diff
@@ -1,15 +1,61 @@
+import time
+import webbrowser
+from pathlib import Path
+from typing import Optional
+
 import typer
 
-from pyscript import LATEST_PYSCRIPT_VERSION, app, cli, plugins
-from pyscript._generator import create_project
+from pyscript import LATEST_PYSCRIPT_VERSION, app, cli, console, plugins
+from pyscript._generator import create_project, file_to_html, string_to_html
+
+
+def wrap_file(title, output, command, show, app_or_file_name, pyscript_version):
+    title = title or "PyScript App"
+
+    # Derive the output path if it is not provided
+    remove_output = False
+    if output is None:
+        if command and show:
+            output = Path("pyscript_tmp.html")
+            remove_output = True
+        elif not command:
+            assert app_or_file_name is not None
+            output = app_or_file_name.with_suffix(".html")
+        else:
+            raise cli.Abort("Must provide an output file or use `--show` option")
+    if app_or_file_name is not None:
+        file_to_html(
+            app_or_file_name,
+            title,
+            output,
+            template_name="wrap.html",
+            pyscript_version=pyscript_version,
+        )
+    if command:
+        string_to_html(
+            command,
+            title,
+            output,
+            template_name="wrap.html",
+            pyscript_version=pyscript_version,
+        )
+    if output:
+        if show:
+            console.print("Opening in web browser!")
+            webbrowser.open(f"file://{output.resolve()}")
```

Comment:
-1 on this for the reasons above

Reply:
Yeah, I think we should remove `--show` from here altogether.

Once again, this is the reminiscent of the past of the `pyscript wrap` functionality.

Reply:
Removed `show` altogether.